ports/ziiaol/Zelda II Enhanced.sh

```diff
@@ -21,15 +21,11 @@ get_controls
 GAMEDIR="/$directory/ports/ziiaol"
 
 # Exports
-export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs32:$GAMEDIR/libs":$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/lib:$GAMEDIR/libs:$LD_LIBRARY_PATH"
```

Comment:
Isn't this game armhf? /usr/lib points to aarch64 libraries.

Reply:
For the patcher. It still boots and runs.

Reply:
Does the Patcher need this `/usr/lib` to be able to play the background music or something like this ? Or it totally fails without ?

Reply:
I have verified this is not needed if $LD_LIBRARY_PATH is present.